src/HexagonAlignLoads.h

```diff
@@ -0,0 +1,22 @@
+#ifndef HALIDE_HEXAGON_ALIGN_LOADS_H
+#define HALIDE_HEXAGON_ALIGN_LOADS_H
```

Comment:
Just wanted to constrain the scope in the beginning. Biggest motivation was that this makes us to go beyond the image on both sides and on HVX if the vector load is unaligned this is ok. May not be the case with other architectures.